cmd/sriovdp/manager.go

```diff
@@ -31,7 +31,9 @@ import (
 
 const (
 	socketSuffix = "sock"
-	netClass     = 0x02 // Device class - Network controller.	 ref: https://pci-ids.ucw.cz/read/PD/02 (for Sub-Classes)
+	netClass     = 0x02   // Device class - Network controller.	 ref: https://pci-ids.ucw.cz/read/PD/02 (for Sub-Classes)
+	accClass     = 0x12   // Device class - Processing accelerator
+	rsuDevice    = 0x0b30 // RSU device - only used for programming the FPGA
```

Comment:
What does "RSU" mean in this context, and do we need to document that somewhere?